src/Microsoft.DotNet.ProjectModel/ProjectReader.cs

```diff
@@ -132,6 +132,37 @@ public Project ReadProject(Stream stream, string projectName, string projectPath
                     throw new FormatException("The assembly file version is invalid: " + fileVersion, ex);
                 }
             }
+            
+            var analyzerOptionsJson = rawProject.Value("analyzerOptions") as JsonObject;
+            if (analyzerOptionsJson != null)
+            {
+                var analyzerOptions = new AnalyzerOptions();
+                
+                foreach (var key in analyzerOptionsJson.Keys)
+                {
+                    switch (key)
+                    {
+                        case "languageId":
```

Comment:
But the languageId here really denotes the language that the project is written in - right? Shouldn't it be in a more general place than analyzeroptions? Since the project is a C# project, the analyzers that can analyze C# will run against it but it feels odd to have this be an analyzer option - Should this be in "compilationOptions" instead?